docs/configuration.md

```diff
@@ -753,7 +753,7 @@ interpretation of the options.
       no
     </td>
     <td>
-      Duration for which a signed URL should be valid.
+      Duration in minutes for which a signed URL should be valid.  When omitted, this defaults to 20 minutes.
```

Comment:
The time unit needs to be specified here. If units aren't recognized, it is a bug. There should be nothing that recoginizes minutes as a default without declaring it.